CefSharp.Core/Internals/RenderClientAdapter.h

```diff
@@ -174,7 +174,7 @@ namespace CefSharp
             virtual DECL void OnCursorChange(CefRefPtr<CefBrowser> browser, CefCursorHandle cursor, CursorType type,
                 const CefCursorInfo& custom_cursor_info) OVERRIDE
             {
-                _renderWebBrowser->SetCursor((IntPtr)cursor);
+				_renderWebBrowser->SetCursor((IntPtr)cursor, (CefSharp::CefCursorType)type);
```

Comment:
I've been using Intellij for so long. I had forgotten that the tab key inserts a tab character sometimes.